src/main/io/osd.c

```diff
@@ -847,7 +865,7 @@ static bool osdDrawSingleElement(uint8_t item)
         return true;
 
     case OSD_CURRENT_DRAW:
-        buff[0] = SYM_AMP;
+        buff[0] = 'A';
```

Comment:
There are capital letters 'V' and 'W' for Volts (battery) and Watts (power), but special smaller symbol for Amperes. Do you think that dedicated symbols for 'w' and 'v' is a better way?

Reply:
Look, for example, at this picture from the current OSD.

https://user-images.githubusercontent.com/4030264/39306943-d16285ac-4961-11e8-87ee-23c8a3d68181.jpg

Using just a normal letter makes the indicator difficult to spot (like the watts indicator on the right).  The reason we have a `W` right now is because no one has drawn one, but we should keep the ones we have. Please, change that back to SYM_AMP.

Reply:
Absolutely agree with you. Your picture looks very convincing.
Just tried to draw a symbol by myself and made sure that numbers are more noticeable and distinguishable with a smaller indicator.
![osd](https://user-images.githubusercontent.com/8948963/39332429-06bc3b5a-49b0-11e8-978b-5fe826eff8c3.png)
I've learnt how to use 'max7456tool'. May I add this symbol to mcm-fonts?
Should I do a separate PR for that or may I push it here?

Reply:
That looks perfect. Thank you so much! Can you submit it as a separate PR? Make one for the configurator with the PNGs and the updated `.mcm` files. Then another one for INAV updating the SYM_WATT value in `src/main/drivers/max7456_symbols.h`. When adding the character, please try to fill one of the empty slots in the middle rather than appending it at the end (note that the one for ` ` is intentionally blank and the 1st one has to be blank per max7456 requirements, the rest are all usable).